modules/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3BlobStore.java

```diff
@@ -160,7 +161,7 @@ public final void collectMetrics(Request<?> request, Response<?> response) {
             final AWSRequestMetrics awsRequestMetrics = request.getAWSRequestMetrics();
             final TimingInfo timingInfo = awsRequestMetrics.getTimingInfo();
             final long requestCount = getCountForMetric(timingInfo, AWSRequestMetrics.Field.RequestCount);
-            final long exceptionCount = getCountForMetric(timingInfo, AWSRequestMetrics.Field.Exception);
+            long exceptionCount = getCountForMetric(timingInfo, AWSRequestMetrics.Field.Exception);
```

Comment:
I think we can revert this change since we no longer take rangeNotSatisfied error away from it.